.github/workflows/publish-bundle.sh

```diff
@@ -0,0 +1,18 @@
+set -ex
+
+BUNDLE_NAME=$(yq e '.name' bundle.yaml)
+echo Found BUNDLE_NAME=$BUNDLE_NAME
+
+charmcraft pack | tee publishlog_charmcraft_pack.log
+
+# Parse name of bundle file from above output
+BUNDLE_FILE=$(cat publishlog_charmcraft_pack.log | cut -d "'" -f 2)
+echo Found BUNDLE_FILE=$BUNDLE_FILE
+
+charmcraft upload $BUNDLE_FILE | tee publishlog_charmcraft_upload.log
+
+# Parse revision number from upload output
+REVISION=$(cat publishlog_charmcraft_upload.log | cut -d " " -f 2)
+echo Found REVISION=$REVISION
+
+charmcraft release $BUNDLE_NAME --revision $REVISION --channel=edge
```

Comment:
I'm not entirely sure if there's a distinction between `latest/edge` and just `edge`. If they are the same, ignore this comment.

Reply:
I'm not sure either.  I worried there might be, so I put it in...